### PR TITLE
SSB: Add cleann

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: data/mods/ssb/statuses.ts [hunk 1]
```diff
@@ -69,6 +69,23 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
...
+			this.add(`c|${getName('cleann')}|Mission Success`);
+		},
+		onFaint() {
+			this.add(`c|${getName('cleann')}|Shazam! They've been destroyed from the inside...`);
+		},
+		onTryMove(attacker, defender, move) {
+			if (move.id === 'maliciousintent') {
+				this.add(`c|${getName('cleann')}|Your team's information has been LEAKED!`);
+			}
+		},
```
Comment (KrisXV) [inline data/mods/ssb/statuses.ts]:
```suggestion
```

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -110,6 +110,41 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Shadow Bone', target);
+		},
+		self: {
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts:130]: why is this in self?

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -110,6 +110,41 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+			onHit(target, source) {
+				let mons = [];
+				for (const foe of target.side.foe.pokemon) {
+					if (foe.m.identified) continue;
+					mons.push(foe);
+				}
+				if (mons.length < 1) return;
+				let randNo = this.random(mons.length);
+				this.add('-message', `${source.name} identified that the opposing team has ${mons[randNo].species}!`);
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts:139]:
```suggestion
				this.add('-message', `${source.name} identified that the opposing team has ${mons[randNo].species}!`);
				this.add(`c|${getName('cleann')}|Your team's information has been LEAKED!`);
```

File: data/mods/ssb/moves.ts (hunk 2 above)
Comment (ghost) [reply data/mods/ssb/moves.ts:130]: So that even if the opponent has a sub, it would still activate

Comment (KrisXV) [reply data/mods/ssb/moves.ts:130]: oh ok

**Pull request merged**